Tools/_Starlight/prepare_test_shards.py

```
@@ -0,0 +1,69 @@
+    )
+
+    # Grab the names of all tests
+    print("Generating shard filters...", file=sys.stderr)
+    list_result = subprocess.run(
+        [
+            "dotnet", "test",
+            "--list-tests",
+            "--no-build",
+            "--configuration", "DebugOpt",
+            "Content.IntegrationTests/Content.IntegrationTests.csproj",
+        ],
```

Ruff (0.15.18)

[error] Starting a process with a partial executable path

(S607)